### ISSUE TITLE
Debian bullseye kde_b38 - Rock 4 SE

### ISSUE THREAD

**Issue by webfinder42** [radxa-build/rock-4se#8]
I have done a fresh installation of this image: rock-4se_debian_bullseye_kde_b38.img.xz and ran an update through rsetup -> update

- Pressed Enter for all defaults during update

I get the login screen after reboot with KDE and can login with my user/password but KDE doesn't load the desktop? it is just completely black screen and can move the mouse around. (i can see the mouse)

What's wrong? and is there a fix?

**Comment by RadxaYuntian** [comment]
The default is to NOT update the existing configuration files. The right answer really should be DO update the existing configuration files unless you modified them, then in that case compare the 2 and include the changes made by us.

**Comment by webfinder42** [comment]
I'm sorry if I didn't express myself clearly enough but that is also what I did "No" to update the existing configuration files.

login screen with KDE works, but when I login with the user blackscreen/No desktop.

Do you want me to do this again on a new fresh installation and try again?

**Comment by RadxaYuntian** [comment]
Yes and you will need to select Y to update the config files when apt asks you.

**Comment by webfinder42** [comment]
So I did it all one more time, fresh install on new SD card.
- Booted into KDE started rsetup -> ran the update from the program
- Selected Yes to update config files when apt asked me [its a loong waiting time]

Now it does even load the desktop, its only on boot menu.
All is OK down the this line.. remember it does not even start KDE desktop now.

[ OK ] Listening on Load/Save RF Kill Switch Status /dev/rfkill Watch. 
Starting Hold until boot process finishes up... [ Starting Terminate Plymouth Boot Screen... 
[ OK ] Started Simple Desktop Display Manager. 
[FAILED] Failed to start dnsmasq - A lightweight DHCP and caching DNS server. 
[ 114.4499871 handlers: 
[ 114.4276751 irq 80: nobody cared (try booting with the "irqpoll" option) 114.471290] 
[<00000000cZ39804c>] irq_default_primary_handler threaded 
[<0000000096bea38d>] regmap_irq_thr [ 
[ 114.4932731 Disabling IRQ #80

**Comment by RadxaYuntian** [comment]
I don't think the system needs either users. Locale might change some commands' output which might be more likely.

Can you check if both conditions must exist for it to fail to boot?

**Comment by webfinder42** [comment]
Okay I will try, I just have one question regarding before.txt, it only runs on first boot?

**Comment by RadxaYuntian** [comment]
Yes. It will be deleted after the system is up.
